wordpress-mobile/gutenberg-mobile
# Fix replacing empty paragraph in inner block

Fixes #1974 

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/20653

To test:
- Open an empty editor w/o any block in it.
- Add Cover/Group block
- Add empty paragraph inside the created block
- Make sure that the created paragraph is empty and selected
- Click on the "Add block" button and select some block
- The paragraph should be replaced by selected block
- PR submission checklist:

Before | After
---|---
![cover-issue](https://user-images.githubusercontent.com/16336501/75979182-40bc7800-5ee0-11ea-9ac0-12373a94ee40.gif) |  ![cover-issue-after](https://user-images.githubusercontent.com/16336501/75979395-b45e8500-5ee0-11ea-9173-74bc36bed27f.gif)
